paper.tex

```diff
@@ -493,4 +503,53 @@ \section{Future work}
 \bibliographystyle{ACM-Reference-Format}
 \bibliography{paper}
 
+\appendix
+
+\section{Geometric Hermite interpolation for Euler spiral}
+\label{appendix:gh}
+
+This appendix gives the detailed algorithm for Geometric Hermite interpolation, determining Euler spiral segment parameters given the tangent angles at the endpoints relative to the chord.
+
+The Euler spiral is represented by $\kappa(s) = k_0 + k_1s$, where $s$ ranges from 0 to 1, ie the arc length is unit normalized. This segment is scaled, rotated, and translated into place so that its endpoints match the desired locations. Immediately, $k_0$ can be determined as $\theta_0 + \theta_1$. Therefore, the remaining parameters to compute are $k_1$ and the ratio of chord length to arc length. The latter \emph{could} be computed from $k_0$ and $k_1$ by numerical integratoiun, but it is more efficient to obtain it at the same time as $k_1$. Setting $k = \theta_0 + \theta_1$ and $\Delta = \theta_1 - \theta_0$, we have:
```

Comment:
s/integratoiun/integration/

P.S.: My brain wants to pronounce this typo in French.